src/content/blog/ddev-on-linux-in-10-minutes.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "DDEV on Linux in 10 Minutes"
```

Comment:
Suggestion from @gitressa 
```suggestion
title: "Install DDEV and Drupal 11 in Linux and create an Xdebug breakpoint with PhpStorm in 10 Minutes (2025)"
```

Reply:
I disagree, because it will make it hard to find the video for those looking for that info ... it's basic white hat SEO.

Could this work?

"Install DDEV in Linux and add PhpStorm Xdebug breakpoint in Drupal (2025)"

https://www.youtube.com/results?search_query=ddev+phpstorm+breakpoint